inlong-sort/sort-flink/base/src/main/java/org/apache/inlong/sort/base/metric/SourceMetricData.java

```diff
@@ -242,6 +243,11 @@ public void outputMetricsWithEstimate(Object data) {
         outputMetrics(1, getDataSize(data));
     }
 
+    public void outputMetricsWithEstimate(Object[] records) {
```

Comment:
maybe we need to specify a data time when sending audit metrics using the metadata "inlong_data_time"

Reply:
Fixed.

Reply:
Maybe we neen to use outputMetricsWithEstimate(record, "inlong_data_time") here ?

Reply:
I will fix it in the next pr.